algoliasearch/client.py

```diff
@@ -252,6 +252,18 @@ def multiple_queries(self, queries,
         data = {'requests': requests, 'strategy': strategy}
         return self._req(True, path, 'POST', request_options, data=data)
 
+    def multiple_get_objects(self, requests, request_options=None):
+        """Send a get objects targeting multiple indices."""
+        if isinstance(requests, (list, tuple)):
+            requests = {'requests': requests}
+
+        path = '/1/indexes/*/objects'
+        return self._req(False, path, 'POST', request_options, data=requests)
+
+    def multiple_batch_objects(self, operations, request_options=None):
```

Comment:
Not sure if we should go with `multiple_batch` or `multiple_batch_objects` 🤔

Reply:
multiple_batch is better ⭐️ 
```suggestion
    def multiple_batch(self, operations, request_options=None):
```